### PR TITLE
Release 0.14.5

### PR DESCRIPTION
**Pull request by vg-coder**
This PR aims at updating the code snippets in the getting-started-java.md to be able to compile them with the 0.14.5 version of the bitcoinj library.

### CODE REVIEW

File: getting-started-java.md [hunk 1]
```diff
@@ -199,12 +197,11 @@ In some cases bitcoinj can generate a large number of events very fast, this is
 ## Receiving money
 
 {% highlight java %}
-kit.wallet().addEventListener(new AbstractWalletEventListener() {
+kit.wallet().addCoinsReceivedEventListener(new WalletCoinsReceivedEventListener() {
     @Override
     public void onCoinsReceived(Wallet w, Transaction tx, Coin prevBalance, Coin newBalance) {
         // Runs in the dedicated "user thread".
-        //
-        // The transaction "tx" can either be pending, or included into a block (we didn't see the broadcast).
```
Comment (schildbach) [inline getting-started-java.md]: Is there a reason for removing this comment?

File: getting-started-java.md [hunk 2]
```diff
@@ -214,16 +211,17 @@ kit.wallet().addEventListener(new AbstractWalletEventListener() {
...
-            public void onSuccess(Transaction result) {
+            public void onSuccess(TransactionConfidence result) {
                 // "result" here is the same as "tx" above, but we use it anyway for clarity.
                 forwardCoins(result);
             }
 
             @Override
             public void onFailure(Throwable t) {}
         });
+
```
Comment (schildbach) [inline getting-started-java.md]: I'd rather not add this empty line.

File: getting-started-java.md (hunk 1 above)
Comment (vg-coder) [reply getting-started-java.md]: You are right. I removed the comment accidentally.

File: getting-started-java.md (hunk 2 above)
Comment (vg-coder) [reply getting-started-java.md]: Yep, let's remove the empty line. 
Now I'm a bit lost of the next step to do, this is my first time contributing to an open source project :). 
Do I have to change your suggestions in my code and commit again?

**Comment by schildbach** [comment]
You can simply add another commit on top and push your branch again. I'll squash all your commits when merging.

**Comment by schildbach** [comment]
Merged, thanks again!

**Pull request closed without merging**